src/main/webapp/js/datahandlers/collegepagedatahandler.js

```diff
@@ -12,96 +12,117 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** @fileoverview This class handles populating the college page view with data from the servlet. */
+/**
+ * @fileoverview This class handles populating the college page view with data
+ * from the servlet.
+ */
 
 goog.module('datahandlers.collegepage');
+
 const {collegepage} = goog.require('finscholar.collegepageview.templates');
-const {ErrorPageView} = goog.require('finscholar.errorpageview');
 
 /** This constant is the endpoint to send a fetch request to. */
 const COLLEGE_SERVLET_ENDPOINT = '/college-data';
 const ERROR_MESSAGE_JSON_UNDEFINED = 'JSON object undefined.';
-const OCCURRENCE_JSON_UNDEFINED = 'A network error has occurred. Failed to load college data.';
-const ACTION_JSON_UNDEFINED = 'Please reload the page or select a different college.';
-const OCCURRENCE_SERVER_ERROR = 'A database error occurred. Failed to render college data.';
-const ACTION_SERVER_ERROR = 'The database failed to retrieve the college. \
-      This college may not exist. Please reload the page or select a different college.';
+const OCCURRENCE_JSON_UNDEFINED =
+    'A network error has occurred. Failed to load college data.';
+const ACTION_JSON_UNDEFINED =
+    'Please reload the page or select a different college.';
+const OCCURRENCE_SERVER_ERROR =
+    'A database error occurred. Failed to render college data.';
+const ACTION_SERVER_ERROR =
+    'The database failed to retrieve the college. This college may not ' +
+    'exist. Please reload the page or select a different college.';
 
 
 /**
- * This method converts from JSON to a JS object map, 
+ * @typedef {{
+ *   schoolName: string,
+ *   institutionType: string,
+ *   acceptanceRate: number,
+ *   averageACTScore: number,
+ *   totalCostAttendance: number,
+ *   netCostForFirstQuintile: number,
+ *   netCostForSecondQuintile: number,
+ *   netCostForThirdQuintile: number,
+ *   netCostForFourthQuintile: number,
+ *   netCostForFifthQuintile: number,
+ *   cumulativeMedianDebt: number
+ * }}
+ */
+let CollegeObject;
+
+/**
+ * This method converts from JSON to a JS object map,
  *  which will be used to populate the college page soy template.
- * @param {*} json - The JSON object to be converted.
- * @return {Object} - The object map representing a college's data.
+ * @param {!CollegeObject|undefined} json - The JSON object to be converted.
```

Comment:
Doesn't make sense to have a non-nullable or undefined parameter here. Either make this nullable or remove the "|undefined"